docs/dotnet/dotnet-http-metrics.md

```diff
@@ -0,0 +1,257 @@
+# Semantic Conventions for HTTP-relevant metrics emitted by .NET and ASP.NET Core
+
+**Status**: [Experimental][DocumentStatus]
+
+This document defines semantic conventions for HTTP metrics emitted by .NET components and runtime.
+
+**Disclaimer:** These are initial .NET metric instruments available in .NET 8 but more may be added in the future.
+
+<!-- toc -->
+
+- [HTTP client](#http-client)
+  * [Metric: `http.client.connections.usage`](#metric-httpclientconnectionsusage)
+  * [Metric: `http.client.connection.duration`](#metric-httpclientconnectionduration)
+  * [Metric: `http.client.duration`](#metric-httpclientduration)
+  * [Metric: `http.client.active_requests`](#metric-httpclientactive_requests)
+  * [Metric: `http.client.failed_requests`](#metric-httpclientfailed_requests)
+- [HTTP server](#http-server)
+  * [Metric: `http.server.duration`](#metric-httpserverduration)
+  * [Metric: `http.server.active_requests`](#metric-httpserveractive_requests)
+  * [Metric: `http.server.unhandled_requests`](#metric-httpserverunhandled_requests)
+
+<!-- tocstop -->
+
+## HTTP client
```

Comment:
`Http.client.request.time_in_queue` is missing.

Reply:
great catch! added